packages/terra-notification-dialog/src/terra-dev-site/doc/example/NoVariantNotificationDialog.jsx

```diff
@@ -33,15 +33,17 @@ class NoVariantNotificationDialog extends React.Component {
           header="No Variant for this dialog"
           isOpen={this.state.isOpen}
           title="Make sure that the title relates directly to the choices."
-          message="The Main Instruction is text used to provide more detail or define terminology. Don’t repeat the title verbatim."
-          primaryAction={{
+          startMessage="The Main Instruction is text used to provide more detail or define terminology. Don’t repeat the title verbatim."
+          acceptAction={{
             text: 'OK',
```

Comment:
Should this text be 'Confirm'?

Reply:
It should be, fixed https://github.com/cerner/terra-framework/pull/899/commits/f95efef89b73c14e91d1b18e9543cd665f7a6191#diff-0e28c13cd31872892958e0d58d16009cR38